### PR TITLE
docs: Fix pydantic example in python-types.md

### PR DESCRIPTION
**Pull request by yaegassy**
First of all,
Thank you for this awesome framework and documentation! 

I fixed the mypy error in the pydantic example.

### CODE REVIEW

**Comment by tiangolo** [comment]
I'm glad you're liking it! :tada: 

Good point, thanks! :heavy_check_mark: 

I took the opportunity to update those docs and include some info about `Optional` and update a bit the terms too.

Thanks for your contribution! :rocket: :cake:

File: docs_src/python_types/tutorial010.py [hunk 1]
```diff
@@ -1,23 +1,7 @@
-from datetime import datetime
-from typing import List, Optional
```
Comment (Liliyord) [inline docs_src/python_types/tutorial010.py]: docs_src/python_types/tutorial010.py

**Review by Liliyord — COMMENTED** [review]
docs_src/python_types/tutorial010.py

File: docs_src/python_types/tutorial010.py (hunk 1 above)
Comment (Liliyord) [reply docs_src/python_types/tutorial010.py]: [Uploading gradle-enterprise-build-validation-scripts-1.0.zip…]()

**Comment by Liliyord** [comment]
[Uploading hypermail-2.4.0.zip…]()

**Review by Liliyord — COMMENTED** [review]
yaegassy:fix-pydantic-example-add-optional

**Pull request merged**